web-console/future-openshift/custom-catalogs/custom-catalogs.md

```diff
@@ -0,0 +1,69 @@
+# Adding custom catalogs to OperatorHub
+
+Admins can create custom catalogs in order to give admin console users the ability to add operators from other sources, beyond the default OperatorHub catalog sources.
+
+## Global Configuration
+
+OperatorHub sources are found through: Administation > cluster settings > global configuration > OperatorHub / sources tab.
+
+![global config](img/operatorhub-custom-catalog1.png)
+
+![OperatorHub config](img/operatorhub-custom-catalog2.png)
+
+## Sources
+
+In the OperatorHub configuration, users can see the current default sources that are feeding into OperatorHub. These sources populate OperatorHub and keep installed operators updated.
+
+![OperatorHub sources default](img/operatorhub-custom-catalog3.png)
```

Comment:
@rileyhuston this mock is a bit confusing to me - might be because I haven't looked at the Hub stuff lately.  Do we only have a single OperatorHub instance ?  It looks like the page title is called Cluster - can we have multiple instances or is this just showing that you can multiple catalog sources for the operator hub?

Reply:
I think my confusion is that there's a breadcrumb "Operator Hubs>... details"
Based on my understanding there's only one Operator Hub, and if that's the case, is there a need to have the breadcrumb?